Makefile

```diff
@@ -4,7 +4,7 @@ test:
 	python3 -m unittest discover --verbose tests/
 
 lint:
-	flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
+	flake8 . --count --max-complexity=10 --max-line-length=152 --statistics
```

Comment:
please keep the original value.
if you have a long string, you can break it to two lines; see for example how it's done in test_guest.py:

https://github.com/IBM/sev-snp-measure/blob/63ae739b356cc00bf20fb1e13129ee16f25a4787/tests/test_guest.py#L23-L26